src/language/mongoDBService.ts

```diff
@@ -211,7 +211,7 @@ export default class MongoDBService {
             );
           }
 
-          worker.terminate().then(() => {
+          void worker.terminate().then(() => {
```

Comment:
what happens if this fails?

Reply:
It is inside try/catch, do you think some extra check should be done here?

Reply:
that try/catch would not necessarily catch a rejection in terminate

Reply:
I see this is in a big promise, but a few lines below there is precedent for `await worker.terminate();` and then resolving the promise.

try/catch won't catch this because you only call .then() and not .catch(). it will only catch if you're using await. Without await you're on your own and have to catch, basically. So if terminate rejects you'll have an unhandled rejection.